index.html

```diff
@@ -236,7 +236,17 @@
     renderSchedule(globalDate, teacherObj, "teacher")
   }
   function renderStudent(studentObj) {
-    console.log(studentObj);
+    var name = studentObj.firstname + " " + studentObj.lastname;
+    var email = studentObj.firstname[0] + studentObj.lastname + "@eastsideprep.org";
```

Comment:
Nope. StudentObj currently looks like this:
![image](https://cloud.githubusercontent.com/assets/3069006/9726135/af0d0734-55a3-11e5-9668-5209b652425e.png)